---
 ssh-agent really needs to be handled by xinitrc like other agents.
 Gentoo bug: 220603

 config/Xsession.in |    8 --------
 1 file changed, 8 deletions(-)

Index: b/config/Xsession.in
===================================================================
--- a/config/Xsession.in
+++ b/config/Xsession.in
@@ -213,14 +213,6 @@ if [ -d /etc/X11/xinit/xinitrc.d ]; then
     done
 fi
 
-# add ssh-agent if found
-sshagent="`gdmwhich ssh-agent`"
-if [ -n "$sshagent" ] && [ -x "$sshagent" ] && [ -z "$SSH_AUTH_SOCK" ]; then
-    command="$sshagent -- $command"
-elif [ -z "$sshagent" ] ; then
-    echo "$0: ssh-agent not found!"
-fi
-
 echo "$0: Setup done, will execute: $command"
 
 eval exec $command
